app/controllers/users_controller.rb

```diff
@@ -62,6 +62,64 @@ def filters
     end
   end
 
+  def change_role
```

Comment:
role changing shouldn't be a separate module, on the edit page, give a dropdown, which shows Staff/admin as options.
Moreover, on users listing page, next to add new user, only Account owner, should see a button that says, Change Owner. when clicked on it, a page appears, which has a dropdown, listing all the emails in the company, account owner can select any user and hit make owner, that should change the owner of the company.